AzureStack/2015-04-01/swagger/GalleryAdmin.json

```diff
@@ -0,0 +1,502 @@
+          "readOnly": true
+        },
+        "Publisher": {
+          "type": "string",
+          "description": "The publisher of the plan",
+          "readOnly": true
+        }
+      }
+    },
+    "ResourceSku": {
+      "type": "object",
+      "description": "The resource sku object",
+      "properties": {
+        "Name": {
+          "type": "string",
+          "description": "The name of the sku",
+          "readOnly": true
+        },
+        "Tier": {
+          "type": "string",
+          "description": "The tier of the sku",
+          "readOnly": true
+        },
+        "Size": {
+          "type": "string",
+          "description": "The size of the sku",
+          "readOnly": true
+        },
+        "Family": {
+          "type": "string",
+          "description": "The family of the sku",
+          "readOnly": true
+        },
+        "Capacity": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The capacity of the sku",
+          "readOnly": true
+        }
+      }
+    },
+    "ResourceDefinition": {
+      "type": "object",
+      "description": "The resource definition object of the azure resource manager",
+      "properties": {
+        "Id": {
+          "type": "string",
+          "description": "The ID of the resource"
+        },
+        "Name": {
+          "type": "string",
+          "description": "The name of the resource",
+          "readOnly": true
+        },
+        "Type": {
+          "type": "string",
+          "description": "The type of the resource",
+          "readOnly": true
+        },
+        "Location": {
+          "type": "string",
+          "description": "The location of the resource"
+        },
+        "Plan": {
+          "$ref": "#/definitions/ResourcePlan",
+          "description": "The plan of the resource"
+        },
+        "Sku": {
+          "$ref": "#/definitions/ResourceSku",
+          "description": "The sku of the resource"
+        },
+        "Kind": {
+          "type": "string",
+          "description": "The kind of the resource"
+        },
+        "Tags": {
+          "type": "object",
+          "description": "The tags on the resource",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "etag": {
+          "type": "string",
+          "description": "The etag of the resource"
+        },
+        "Properties": {
+          "type": "object",
+          "description": "The custom properties of the resource"
+        }
+      },
+      "x-ms-azure-resource": true
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Get subscription credentials which uniquely identify Azure stack subscription. The subscription ID forms part of the URI for every service call."
```

Comment:
Keep it short and sweet: "The Azure SubscriptionId that uniquely identifies a subscription."